geocity/apps/submissions/views.py

```diff
@@ -544,6 +592,26 @@ def handle_form_submission(self, form, action):
             return self.handle_complementary_documents_form_submission(form)
         elif action == models.ACTION_REQUEST_INQUIRY:
             return self.handle_request_inquiry_form_submission(form)
+        elif action == models.ACTION_CREATE_SERVICE_FEE:
+            return self.handle_service_fee_form_submission(form)
+        elif action == models.ACTION_UPDATE_SERVICE_FEE:
+            return self.handle_service_fee_form_submission(form)
+        elif action == models.ACTION_DELETE_SERVICE_FEE:
+            return self.handle_service_fee_form_submission(form)
```

Comment:
Do you really need to discriminate among CRUD action here ?

Reply:
Outdated.